examples/tokenvm/avalanche-ops.aws.md

```diff
@@ -0,0 +1,249 @@
+## Deploy TokenVM in custom network with avalanche-ops
+
+[avalanche-ops](https://github.com/ava-labs/avalanche-ops) provides a command-line interface to set up nodes and install custom VMs with the support for custom networks.
+
+### Step 1
+
+Install [Rust](https://www.rust-lang.org/tools/install) to compile avalanche-ops, or download from the [latest release page](https://github.com/ava-labs/avalanche-ops/releases/tag/latest):
+
+```bash
+git clone git@github.com:ava-labs/avalanche-ops.git
+cd ./avalanche-ops
+./scripts/build.release.sh
+./target/release/avalancheup-aws --help
+```
+
+```bash
+# to download on Mac
+export LINUX_ARCH_TYPE=$(uname -m)
+echo ${LINUX_ARCH_TYPE}
+rm -f ./avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin
+wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin"
+mv ./avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin /tmp/avalancheup-aws
+chmod +x /tmp/avalancheup-aws
+/tmp/avalancheup-aws --help
+```
+
+```bash
+# to download on Linux
+export LINUX_ARCH_TYPE=$(uname -m)
+echo ${LINUX_ARCH_TYPE}
+rm -f ./avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu
+wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu"
+mv ./avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu /tmp/avalancheup-aws
+chmod +x /tmp/avalancheup-aws
+/tmp/avalancheup-aws --help
+```
+
+### Step 2
+
+Now we can spin up a new network of 6 nodes with some defaults:
+- `avalanche-ops` supports [Graviton-based processors](https://aws.amazon.com/ec2/graviton/) (ARM64). Use `--arch-type arm64` to run nodes in ARM64 CPUs.
+- `avalanche-ops` supports [EC2 Spot instances](https://aws.amazon.com/ec2/spot/) for cost savings. Use `--instance-mode=spot` to run instances in spot mode.
+
+```bash
+# launch/download all artifacts for x86 platform
+avalancheup-aws default-spec \
+--arch-type amd64 \
+--rust-os-type ubuntu20.04 \
```

Comment:
let's bump this to 22.04 for compatibility with how golang builds as of 1.20: https://github.com/golang/go/issues/57328

Reply:
Alternatively, I can change to building on 20.04?